samtranslator/schema/schema.py

```diff
@@ -1,24 +1,31 @@
 from __future__ import annotations
 
-from enum import Enum
 from typing_extensions import Literal
 from typing import Any, Dict, List, Optional, Union
 
-from pydantic import BaseModel as LenientBaseModel
-from pydantic import Extra, Field, constr
```

Comment:
some fields cannot be imported due to https://github.com/aws/serverless-application-model/actions/runs/3501134346/jobs/5864465766